plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -89,17 +97,19 @@ export const OwnershipCard = (props: {
                 placement="top"
                 arrow
                 title={`${
-                  getRelationsType === 'direct' ? 'Direct' : 'Aggregated'
+                  derivedRelationsType === 'direct' ? 'Direct' : 'Aggregated'
                 } Relations`}
               >
                 <Switch
                   color="primary"
-                  checked={getRelationsType !== 'direct'}
-                  onChange={() =>
-                    getRelationsType === 'direct'
-                      ? setRelationsType('aggregated')
-                      : setRelationsType('direct')
-                  }
+                  checked={derivedRelationsType !== 'direct'}
+                  onChange={() => {
+                    const updatedRelationsType =
+                      derivedRelationsType === 'direct'
```

Comment:
Minor thing but I think we can keep the old onChange function that I find a bit more readable.
```
   onChange={() =>
                    getRelationsType === 'direct'
                      ? setRelationsType('aggregated')
                      : setRelationsType('direct')
                  }
```

Alternatively inline it in setRelationsType without the `const`.

Reply:
Yup, I might have overthought this one to be honest :smile: I think I found the `get` a bit confusing as it's not a getter. Will revert back though as I don't really have a strong opinion here :+1:

Reply:
Super minor so no worries :)